src/components/Search/SearchPageHeader.tsx

```diff
@@ -309,6 +309,7 @@ function SearchPageHeader({queryJSON, hash, onSelectDeleteOption, setOfflineModa
                     customText={translate('workspace.common.selected', {selectedNumber: selectedTransactionsKeys.length})}
                     options={headerButtonsOptions}
                     isSplitButton={false}
+                    shouldUseStyleUtilityForAnchorPosition
```

Comment:
Looks like there was conflict on this line. Please retest as `style={styles.ml2}` was removed